project/Build.scala

```diff
@@ -64,7 +64,7 @@ object Scrooge extends Build {
     version := libVersion,
     organization := "com.twitter",
     crossScalaVersions := Seq("2.10.5" , "2.11.7"),
-    scalaVersion := "2.10.5",
+    scalaVersion := "2.11.7",
```

Comment:
General curiousity: why did you have to change scalaVersion? The line above, crossScalaVersions makes me think this is anticipated, but I'm also thinking from the perspective of Twitter guys accepting a pull request down the road.